Add AI Agent Manager feature in preview (#4496)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/AI-Agent-Manager-Now-available-in-Preview.md

---
date: "2026-03-31"

> title: "AI Agent Manager: Now Available in Preview"

Reviewer: ```suggestion
title: "AI Agent Manager now available in Public Preview"
```

product_area: "Application enablement & solutions"
change_type:

>   - value: "change-type-preview"

Reviewer: ```suggestion
  - value: "change-2c7RdTdXo4"
```

    label: "Preview"
component:

>   - value: "component-ai-agent-manager"

Reviewer: ```suggestion
  - value: "component-YbYJ3gLU_"
```

    label: "Web SDK"
build_artifact:

>   - value: "ai-agents"

Reviewer: ```suggestion
  - value: "tc--gWykZ78v"
```

>   - value: "ai-agents"

Reviewer: We do not yet have this build artifact, do we need to add this @janhommes?

Reviewer 2: yes, this is the new name of the build_artifact. Not sure if we need to add it, aren't build artifacts not normally taken as is?

Reviewer: > yes, this is the new name of the build_artifact. Not sure if we need to add it, aren't build artifacts not normally taken as is?

No, we must manually add new build artifacts to the buildartifact.json file. I just did that.

    label: "ai-agents"
ticket: ""
version: "0.0.9"
---
{{< c8y-admon-preview >}}

> This feature is available in Preview. It is intended for evaluation and testing purposes and may be subject to change.

Reviewer: ```suggestion
This feature is in Public Preview, that is, it is not enabled by default and may be subject to change in the future.
```

Reviewer: This is a kind of standard phrase we use in this case.

{{< /c8y-admon-preview >}}

The AI Agent Manager introduces a new capability to create, configure, and run AI-powered agents directly within a tenant through an intuitive user interface.

> Agents can analyze device data and execute actions based on configurable logic. They can be granted access to Cumulocity data such as inventory, events, alarms, and measurements, enabling seamless integration with existing IoT applications without additional setup. Agents can be extended with custom tools or MCP servers, and their behavior can be validated in a sandbox environment before being enabled for real-time automation and complex operational workflows.

Reviewer: ```suggestion
Agents can analyze device data and execute actions based on configurable logic. They can be granted access to {{< product-c8y-iot >}} data such as inventory, events, alarms, and measurements, enabling seamless integration with existing IoT applications without additional setup. Agents can be extended with custom tools or MCP servers, and their behavior can be validated in a sandbox environment before being enabled for real-time automation and complex operational workflows.
```

> Agents can analyze device data and execute actions based on configurable logic. They can be granted access to {{< product-c8y-iot >}} data such as inventory, events, alarms, and measurements, enabling seamless integration with existing IoT applications without additional setup. Agents can be extended with custom tools or MCP servers, and their behavior can be validated in a sandbox environment before being enabled for real-time automation and complex operational workflows.

Reviewer 2: What do you mean by the "sadbox" environment?

Author: I was referring to the chat view in the Admin app, where agents can be tested before being used in other applications. On thinking more about i “sandbox” is not full true , will remove that
